packages/template-retail-react-app/jest-setup.js

```diff
@@ -27,7 +27,7 @@ global.jsdom.reconfigure({url: 'https://www.domain.com'})
 
 configureTestingLibrary({
     // Increase to: 6 x default timeout of 1 second
-    ...(process.env.CI ? {asyncUtilTimeout: 6000} : {})
+    ...(process.env.CI ? {asyncUtilTimeout: 10000} : {})
```

Comment:
It seems like the tests are timeout on checkout lately. Increasing timeout here to see if it still fails too often

Reply:
Do we have a story for revisiting tests? If not, can you add one?